packages/flex-dev-utils/package.json

```diff
@@ -40,9 +40,10 @@
     "axios": "^0.21.1",
     "axios-mock-adapter": "^1.19.0",
     "copy-template-dir": "^1.4.0",
-    "flex-plugins-utils-exception": "^0.19.0",
-    "flex-plugins-utils-logger": "^0.19.0",
-    "flex-plugins-utils-spawn": "^0.19.0",
+    "flex-plugins-utils-exception": "^4.5.0-beta.0",
```

Comment:
i thought you wanted to name them all `flex-plugin-*` 😛

Reply:
Not in this PR - I wil migrate everything over and then change the name of all packages after.